### PR TITLE
🌐 Add Polish translation for `docs/pl/docs/tutorial/index.md`

### PR DESCRIPTION
**Pull request by MKaczkow**
`docs/pl/tutorial/index.md` to polish translation

### CODE REVIEW

File: docs/pl/docs/tutorial/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Samouczek - Instrukcja - Wprowadzenie
```
Comment (rafsaf) [inline docs/pl/docs/tutorial/index.md:1]: I would change to simpler `# Samouczek - Wprowadzenie`, see my comment in `mkdocs.yml`

File: docs/pl/docs/tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    pip install fastapi
+    ```
+
+    Also install `uvicorn` to work as the server:
+
+    ```
+    pip install "uvicorn[standard]"
+    ```
+
+    And the same for each of the optional dependencies that you want to use.
```
Comment (rafsaf) [inline docs/pl/docs/tutorial/index.md:70]: Shoudn't it be in English :trollface: 
Or maybe it's some convention to leave notes alone?

File: docs/pl/docs/tutorial/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+## Instrukcja dla zaawansowanych
+
+Jest też **Instrukcja dla zaawansowanych**, którą możesz przeczytać po lekturze tego **Samouczka - Instrukcji**.
+
+**Instrukcja dla zaawansowanych**, opiera się na tym samouczku, używa tych samych pojęć, żeby pokazać Ci kilka dodatkowych funkcji.
+
+Najpierw przeczytaj **Samouczek - Instrukcja** (czytasz to teraz).
+
+Samouczek jest zaprojektowany tak, że możesz stworzyć kompletną aplikację używając tylko **Samouczka - Instrukcji**, a następnie rozszerzać ją na różne sposoby, w zależności od potrzeb, używając kilku dodatkowych pomysłów z **Instrukcji dla zaawansowanych**
+
```
Comment (rafsaf) [inline docs/pl/docs/tutorial/index.md:81]: Personally, I would change `Instrukcja dla zaawansowanych` na `Zaawansowany poradnik`, it's shorter and sounds better.
Also, as noted in first lines, `Samouczek` alone is better than `Samouczek - Instrukcja` (and it's not really correct in Polish grammar as far as I know, we don't use upper case letters in middle of titles in Polish, we adopted that from English). There are also few mistypes over here

So at the end of day, the fixed part may look like:

```
## Zaawansowany poradnik

Jest też **Zaawansowany poradnik**, który możesz przeczytać po lekturze tego **Samouczka**.

**Zaawansowany poradnik** opiera się na tym samouczku, używa tych samych pojęć, żeby pokazać Ci kilka dodatkowych funkcji.

Najpierw jednak  powinieneś przeczytać **Samouczek** (czytasz go teraz).

Ten rozdział jest zaprojektowany tak, że możesz stworzyć kompletną aplikację używając tylko informacji tutaj zawartych, a następnie rozszerzać ją na różne sposoby, w zależności od potrzeb, używając kilku dodatkowych pomysłów z **Zaawansowanego poradnika**.
```

File: docs/pl/mkdocs.yml [hunk 4]
```diff
@@ -52,6 +52,8 @@ nav:
   - tr: /tr/
   - uk: /uk/
   - zh: /zh/
+- Tutorial - User Guide:
+  - tutorial/index.md
```
Comment (rafsaf) [inline docs/pl/mkdocs.yml:56]: And this is something I would **Definitely** change to 
```yml
- Samouczek:
  - tutorial/index.md
```
We should avoid naming this topic in English, its much better to use Polish here. FastAPI docs are smart enough to fill docs we don't cover in English even if we use Polish name (screenshot), this is probably detected by `tutorial/index.md`.

See current screenshot (I pulled your forked repo and changed code like above).


image

**Review by rafsaf — CHANGES_REQUESTED** [review]
Hi @MKaczkow 

Thanks for providing this PR 👍 
First of all, I've pulled your forked project so it's not only review by looking at changes.

This is kind of "important", because we will decide what translation of topics names `Advanced User Guide` and `Tutorial - User Guide` will be, this can be hardly changed ever in the future.

 I would definitely opt for my last review comment and change in `mkdocs.yml`, I've searched through other languages like PT, they seem to follow `mkdocs.yml` syntax just like I did. Rest of comments are just to consider. Have a good day ;)

**Comment by MKaczkow** [comment]
Hi @rafsaf,
I've applied changes you suggested, hope everything is fine this time.

Thank you for reviewing this PR :-)

**Review by rafsaf — APPROVED** [review]
LGTM :P

Thanks again, now I belive we should wait for one more person (Polish native speaker) to approve it and this can be merged.

**Review by kiwimic — APPROVED** [review]
As polish native speaker I can approve that this polish translation is correct

**Comment by tiangolo** [comment]
Awesome, thanks @MKaczkow ! :cake: 

And thanks for the reviews @kiwimic and and @rafsaf ! :bow:

**Pull request merged**